### PR TITLE
Get articles

### PR DESCRIPTION
**Pull request by ZeusLGR**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -49,3 +49,38 @@ describe("GET /api/topics", () => {
...
+
+describe("GET /api/articles", () => {
+  test("status:200, responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
```
Comment (robert112233) [inline __tests__/app.test.js:61]: The length assertion is really important when your expect statement is inside a forEach, good work 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -49,3 +49,38 @@ describe("GET /api/topics", () => {
...
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+          expect(articles).toBeSortedBy("created_at", { descending: true });
```
Comment (robert112233) [inline __tests__/app.test.js:74]: Good use of toBeSortedBy, I would consider either changing the test description to mention sorting OR if following best practice; create another test that tests the sorting functionality, when using TDD each test should test one piece of functionality.

File: controllers/controllers.js [hunk 3]
```diff
@@ -13,3 +13,13 @@ exports.getTopics = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  selectArticles()
+    .then((articles) => {
+      return res.status(200).send({ articles });
```
Comment (robert112233) [inline controllers/controllers.js:20]: Well done for sending the data back on a semantic key 👍

File: __tests__/app.test.js (hunk 2 above)
Comment (robert112233) [reply __tests__/app.test.js:74]: Not a huge issue though, you can still merge 👍

**Review by robert112233 — APPROVED** [review]
Merge when ready 👍  see feedback for future changes if you have time

File: __tests__/app.test.js (hunk 2 above)
Comment (ZeusLGR) [reply __tests__/app.test.js:74]: Thanks Rob, I'll put this into place now

**Pull request merged**